packages/main/src/Carousel.hbs

```diff
@@ -1,14 +1,21 @@
 <section
     class="ui5-carousel-root"
     tabindex="0"
+    role="listbox"
+    aria-activedescendant="{{ariaActiveDescendant}}"
     @keydown={{_onkeydown}}
 >
     <div class="ui5-carousel-overflow-hidden">
         <div class="{{classes.content}}" style="{{styles.content}}">
             {{#each pages}}
                 <div class="{{../classes.page}}">
                     {{#each this}}
-                        <div class="ui5-carousel-item" style="width: {{this.width}}%">
+                        <div id="carousel-item-{{posinset}}"
```

Comment:
Please also incorporate `{{_id}}` to make it truly unique for IE where the shadow DOM of all components will be in the main HTML document, f.e. `id="{{_id}}-carousel-item-{{posinset}}"`, and respectively the same in `ariaActiveDescendant`

Reply:
done